src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -63,8 +63,16 @@
 
     private AgencyAndId stop;
 
+    /**
+     * The headsign of the alert
+     */
     private String direction;
 
+    /**
+     * Direction of the trip as GTFS route direction
```

Comment:
We should probably make it even more clear that this is the raw direction_id field from GTFS *trips*, which distinguishes between different directions on the same *route*.